main.go

```diff
@@ -288,17 +288,16 @@ func RdColumnsAndPk(leastCol int) ([]ColumnType, []ColumnType) {
 }
 
 func rdColumns(least int) []ColumnType {
-	colCnt = util.RdRange(columnLeast, columnMost)
-	if colCnt < least {
-		colCnt = least
+	colNum := util.RdRange(columnLeast, columnMost)
+	colCnt = colNum
```

Comment:
IIRC, colCnt is a global variable which will be used when generating DMLs. So the value of colCnt may be equal to the count of columns in a table, I mean putting this value assign after the if section(where colNum is finally determinied).